### PR TITLE
feat(log): add exra cmd-flags for file logger

### PR DESCRIPTION
**Pull request by yqlbu**
### Background

Add extra cmd flags to allow end-users to customize settings for file logger.

Ref: https://github.com/natefinch/lumberjack

### Checklist

- [x] The Pull Request has been fully tested
- [ ] There's an entry in the CHANGELOGS
- [ ] There is a user-facing docs PR against https://github.com/juicity/juicity

### Full changelogs

- feat(pkg/log): add options for fileLogger
- feat(common/consts): persist default vals for log pkg
- feat(cmd/{server,client}): add extra flags for fileLogger
- docs(server,client): add extra logger auguments

### Issue reference

NA

### Test Result

Run without `logFileWriter`:

image

Run with `logFileWriter`:

image

### CODE REVIEW

**Comment by mzz2017** [comment]
Does it really work for gliderLog?

**Comment by yqlbu** [comment]
> Does it really work for gliderLog?

It ONLY works for `fileLogWriter`. More precisely, if `--log-enable-file-writer` is NOT passed, the app will ONLY use `ConsoleWriter`.

File: cmd/server/README.md [hunk 1]
```diff
@@ -63,13 +63,7 @@ Full configuration:
...
 ## Arguments
 
-| Name                  | Description                             | Default Value                | Required |
-| --------------------- | --------------------------------------- | ---------------------------- | -------- |
-| `--config, -c`        | specify config file path                | NA                           | yes      |
-| `--disable-timestamp` | disable timestamp                       | false                        | no       |
-| `--log-file`          | write logs to file                      | /var/log/juicity/juicity.log | no       |
-| `--log-disable-color` | disable colorful log output             | false                        | no       |
-| `--log-format`        | specify log format; options: [raw,json] | raw                          | no       |
+Run `juicity-client run -h` to get the full arguments.
```
Comment (mzz2017) [inline cmd/server/README.md:66]:
```suggestion
Run `juicity-server run -h` to get the full arguments.
```

**Comment by yqlbu** [comment]
LGTM, ready to merge?

**Comment by mzz2017** [comment]
Test Result:

image

**Review by mzz2017 — DISMISSED** [review]

**Review by mzz2017 — APPROVED** [review]

**Pull request merged**